Avances Grupo 2

RenatoLarenasMunoz opened the pull request:
avance en funciones en Boss
avance de movimiento en Bonus
avance de movimiento en Gameplay

Zetossover committed:
Avanze Grupo2 Bonus

hotlineseasoningcity committed:
player 2 movement

crosshair movement

hotlineseasoningcity committed:
Merge branch 'main' of https://github.com/hotlineseasoningcity/USEK-Arcade

RenatoLarenasMunoz committed:
BossProgress

Avance Boss

RenatoLarenasMunoz committed:
Merge branch 'main' of https://github.com/hotlineseasoningcity/USEK-Arcade

Zetossover committed:
Coleccionable

Hice una clase para poder crear un coleccionable mediante texto.

RenatoLarenasMunoz committed:
avance Boss

- Boss desaparece al morir
- asocio valores de vida con una imagen cuyo relleno varia dependiendo de la vida

RenatoLarenasMunoz committed:
Merge branch 'main' of https://github.com/hotlineseasoningcity/USEK-Arcade

Zetossover commented:
Fco - Creación de clase de coleccionable mediante uso de texto.
Renato - Asociación de vida al boss y reacción de este mismo.

hotlineseasoningcity committed:
Organization

Improved folder organization

hotlineseasoningcity committed:
Enemy Behaviors

Enemy Behaviors set in Base Class

Zetossover committed:
Generacion de mapa bonus

Creacion de codigo base para ambos niveles bonus

RenatoLarenasMunoz committed:
avance

- comportamiento de escudar aliados implementado (enemyShield)
- Boss activa y desactiva escudo dependiendo de la condición

RenatoLarenasMunoz committed:
Merge branch 'main' of https://github.com/hotlineseasoningcity/USEK-Arcade

RenatoLarenasMunoz committed:
avance

Sprites Boss y enemyShield añadidos

RenatoLarenasMunoz committed:
avance Boss

cambio de sprites dependiendo del estado del Boss (protegido/desprotegido)

RenatoLarenasMunoz committed:
avance

- funcionamiento de spawneo de enemigos listo
-funcionamiento de escudo del Boss listo

hotlineseasoningcity committed:
player 1 + 2 movement + configuring enemies

Added Player movement and started configuring the Enemies' behavior

RenatoLarenasMunoz committed:
cambio posBoss

auto explicativo

Zetossover committed:
Patrullaje de Enemigo

Que el enemigo se mueva mediante puntos aleatorios del mapa.

hotlineseasoningcity committed:
Merge branch 'elwatona:main' into main

RenatoLarenasMunoz committed:
avances

- habilidad del Boss para dañar al Player
- habilidad del Player para dañar enemigos

Zetossover committed:
Radio de enemigo y imagenes

Agregue el sistema de derrota al entrar en el radio del enemigo el jugador pierde automáticamente

dy-0 committed:
new sprites + scripts + fixes

dy-0 committed:
Merge branch 'Gameplay' of https://github.com/hotlineseasoningcity/USEK-Arcade into Gameplay

dy-0 committed:
test

dy-0 committed:
test2

dy-0 committed:
additions

Zetossover committed:
Todo

Fianl

Zetossover committed:
Mira de enemigo

Ahora miran donde van

RenatoLarenasMunoz committed:
avance

Boss ataca

RenatoLarenasMunoz committed:
avance

- setteo de canvas
- cambio nombre funciones

RenatoLarenasMunoz committed:
.

.

Zetossover committed:
Cambio de escena

Cambio de escena en los niveles bonus

Zetossover committed:
Merge branch 'main' of https://github.com/hotlineseasoningcity/USEK-Arcade

Zetossover committed:
Merge branch 'elwatona:main' into main

RenatoLarenasMunoz committed:
cambios menores

.

RenatoLarenasMunoz committed:
Merge branch 'main' of https://github.com/hotlineseasoningcity/USEK-Arcade

Zetossover committed:
Merge branch 'elwatona:main' into main

RenatoLarenasMunoz committed:
avance

- visuales de vida para Boss
- arreglos visuales generales

Zetossover committed:
21-11-2024

Score

dy-0 committed:
test

RenatoLarenasMunoz committed:
.

.

hotlineseasoningcity committed:
arreglar setting del juego

RenatoLarenasMunoz committed:
avance

Player implementado en escena Boss

RenatoLarenasMunoz committed:
Merge branch 'main' of https://github.com/hotlineseasoningcity/USEK-Arcade

RenatoLarenasMunoz committed:
avance

visual Boss cargando ataque

dy-0 committed:
fix enemy shoot + improved movement

RenatoLarenasMunoz committed:
avance

Boss con ataque funcional

Zetossover committed:
BonusLevel2

Creación del nivel 2 del bonus y arreglo de bug de coleccionable

Zetossover committed:
Merge branch 'main' of https://github.com/hotlineseasoningcity/USEK-Arcade

hotlineseasoningcity committed:
enemy behavior fix

hotlineseasoningcity committed:
Merge branch 'main' of https://github.com/hotlineseasoningcity/USEK-Arcade

RenatoLarenasMunoz committed:
.

.

RenatoLarenasMunoz committed:
Merge branch 'main' of https://github.com/hotlineseasoningcity/USEK-Arcade

Zetossover committed:
.

RenatoLarenasMunoz committed:
.

.

RenatoLarenasMunoz committed:
avance

disparo daña al Boss

RenatoLarenasMunoz committed:
avance

barra de vida del Boss funcional

RenatoLarenasMunoz committed:
avance

intento de limitar crosshair

RenatoLarenasMunoz committed:
.

.

RenatoLarenasMunoz committed:
.

.

RenatoLarenasMunoz committed:
Update Controles

Finalmente controles separados

RenatoLarenasMunoz committed:
.

.

dy-0 committed:
se me ha crasheado el pc 3 veces seguidas

dy-0 committed:
health bar + audio

dy-0 committed:
couple changes

RenatoLarenasMunoz committed:
..

.

RenatoLarenasMunoz committed:
.

.

RenatoLarenasMunoz committed:
Revert "."

This reverts commit 767ddbb8ef8cd44b1626e0e4bf9c2b3b1b16a97f.

Zetossover committed:
Bonus level 2

Cambio de enemigos en el bonus 2

Zetossover committed:
Merge branch 'main' of https://github.com/hotlineseasoningcity/USEK-Arcade

Zetossover committed:
Agregado del Menu y Game Over

Se agrego el menu y el game over al juego
